### DISCUSSION TITLE
Can I use this library In Unity ?

### DISCUSSION THREAD

**Discussion by czy77** [icerpc/icerpc-csharp#3958] [Q&A] [answered]

**Comment by pepone** [comment]
Hi @czy77 

IceRPC for C# requires .NET 8.0. I'm not an expert in Unity but it seems like Unity cannot target .NET 8 yet.

**Comment by ReeceHumphreys** [comment, marked as answer]
Hello @czy77, I want to jump in on @pepone's answer. Unity is planning to add .NET 8 support in the future. However, they do not have a timeline yet. Here is a link to a forum post about it: https://forum.unity.com/threads/unity-future-net-development-status.1092205/

> EDIT (October 27, 2023): We're focused internally on developing and shipping new .NET support, with our current target being .NET 8.
